### ISSUE TITLE
image preload

### ISSUE THREAD

**Issue by nosovk** [Valexr/Slidy#68]
In a demo images start to load just after moving to a new slide.
It could be fixed using preload attribute. It could be added to the next slide, before sliding onto it.

**Comment by Valexr** [comment]
What demo & preload attribute do you mean?
Please show me the code... don't understand🤓

**Comment by nosovk** [comment]
https://web.dev/i18n/en/preload-responsive-images/

**Comment by Valexr** [comment]
🤔Create `<link>` tags in `<head>` ?

**Comment by Valexr** [comment]
Oh, @nosovk, maybe you know - why Safari sometimes scratching transform transition? And with all images loaded, too…🤔

**Comment by nosovk** [comment]
You can create link inside slider.
https://svelte.dev/repl/844e2acee1bf4f759f2206c9ae604d99?version=3.46.4

**Comment by Valexr** [comment]
> You can create link inside slider.
> https://svelte.dev/repl/844e2acee1bf4f759f2206c9ae604d99?version=3.46.4

Good point 👍🏻Tx🤓receive this too @slidy/svelte dev👌🏻(I’m working on the @slidy/core). PRs welcome 🤗

**Comment by nosovk** [comment]
https://svelte.dev/repl/844e2acee1bf4f759f2206c9ae604d99?version=3.46.4
more like that

**Comment by nosovk** [comment]
> Oh, @nosovk, maybe you know - why Safari sometimes scratching transform transition? And with all images loaded, too…🤔
> 
> Without images haven’t scratching🤷🏻‍♂️ & `will-change` don’t help

will check later today

**Comment by Valexr** [comment]
> > Oh, @nosovk, maybe you know - why Safari sometimes scratching transform transition? And with all images loaded, too…🤔
> > Without images haven’t scratching🤷🏻‍♂️ & `will-change` don’t help
> 
> will check later today

👌🏻 will wait 

n& I move it into discussion maybe? 

I think it’s not Issue theme…
